src/core/apim_nodo_monitoring_services.tf

```diff
@@ -75,7 +69,10 @@ module "apim_nodo_monitoring_api" {
   })
 
   xml_content = templatefile("./api/nodopagamenti_api/monitoring/v1/_base_policy.xml.tpl", {
-    base-url                  = azurerm_api_management_named_value.default_nodo_backend.value
-    is-nodo-decoupler-enabled = var.apim_nodo_decoupler_enable
+    base-url     = var.env_short == "p" ? "https://{{ip-nodo}}" : "http://{{aks-lb-nexi}}{{base-path-nodo-oncloud}}"
```

Comment:
the logic adopted in the node policies is to always refer to the `default-node-backend` namedValue

https://github.com/pagopa/pagopa-infra/blob/main/src/core/apim.tf#L329